torch/csrc/jit/passes/quantization/insert_quant_dequant.cpp

```diff
@@ -826,9 +826,7 @@ void propagateQParams(
   insertDeQuantForAllUse(graph, quantized_output, quantized_output);
 }
 
-void propagateDequantize(Value* output, const std::vector<Value*> inputs) {
-  Node* n = output->node();
-  Graph* graph = n->owningGraph();
+void removeDequantizeFromInputs(const std::vector<Value*>& inputs) {
```

Comment:
Can you add a test showing the functionality here before you land?

Reply:
this is a refactor, not a new functionality, it's already tested in test_quantize_script.py